src/components/common/CodeReview.js

```diff
@@ -252,6 +262,26 @@ class CodeReview extends Component {
     return comment.type !== COMMENT;
   }
 
+  adaptHeight = () => {
+    // set timeout to leave time for the height to be set
+    setTimeout(() => {
+      const { standalone, isStudentView } = this.props;
+      // adapt height when :
+      // - not in standalone (so when in an iframe)
+      // - is in studentView
+      if (!standalone && isStudentView) {
+        // get height from the root element and add a small margin
+        const actualHeight = this.rootRef.current.scrollHeight + 30;
+        if (window.frameElement) {
+          window.frameElement.style['min-height'] = `${actualHeight}px`;
+          window.frameElement.style.overflowY = 'hidden';
+          window.frameElement.scrolling = 'no';
+          window.frameElement.style.height = `${actualHeight}px`;
+        }
+      }
+    }, 1);
```

Comment:
Factor this number out to constants and make it something less quick, like 50ms or 100ms just in case.